TsRandomizer/Randomisation/ItemLocationMap.cs

```diff
@@ -821,14 +821,14 @@ void AddLanternLocations()
 			Add(new ItemKey(11, 22, 280, 505), "Lab: File Cabinet Staircase Lantern 4", null, UpperLab & LanternCube);
 			Add(new ItemKey(11, 17, 216, 281), "Lab: Trash Stairs Lantern 1", null, MainLab & LanternCube);
 			Add(new ItemKey(11, 17, 120, 505), "Lab: Trash Stairs Lantern 2", null, MainLab & LanternCube);
-			Add(new ItemKey(11, 18, 126, 144), "Lab: Trash Jump Lantern 1", null, MainLab & (SeedOptions.LockKeyAmadeus ? DoubleJumpOfNpc : R.UpwardDash) & LanternCube);
-			Add(new ItemKey(11, 18, 782, 112), "Lab: Trash Jump Lantern 2", null, MainLab & (SeedOptions.LockKeyAmadeus ? DoubleJumpOfNpc : R.UpwardDash) & LanternCube);
+			Add(new ItemKey(11, 18, 126, 144), "Lab: Trash Jump Lantern 1", null, MainLab & R.DoubleJump & LanternCube);
+			Add(new ItemKey(11, 18, 782, 112), "Lab: Trash Jump Lantern 2", null, MainLab & DoubleJumpOfNpc & LanternCube);
 			Add(new ItemKey(11, 19, 120, 169), "Lab: Genza Door Lantern 1", null, UpperLab & LanternCube);
 			Add(new ItemKey(11, 19, 264, 169), "Lab: Genza Door Lantern 2", null, UpperLab & LanternCube);
 			Add(new ItemKey(11, 23, 104, 169), "Lab: Spider Hell Entrance Lantern 1", null, LabResearchWing & R.CardA & LanternCube);
 			Add(new ItemKey(11, 23, 280, 169), "Lab: Spider Hell Entrance Lantern 2", null, LabResearchWing & R.CardA & LanternCube);
-			Add(new ItemKey(11, 3, 1448, 489), "Lab: Lower Trash Lantern 1", null, MainLab & (FloodsFlags.Lab ? R.Free : (SeedOptions.LockKeyAmadeus ? DoubleJumpOfNpc : R.UpwardDash)) & LanternCube);
-			Add(new ItemKey(11, 3, 152, 489), "Lab: Lower Trash Lantern 2", null, MainLab & (FloodsFlags.Lab ? R.Free : (SeedOptions.LockKeyAmadeus ? DoubleJumpOfNpc : R.UpwardDash)) & LanternCube);
+			Add(new ItemKey(11, 3, 1448, 489), "Lab: Lower Trash Lantern 1", null, MainLab & NeedSwimming(FloodsFlags.Lab) & LanternCube);
```

Comment:
turns out both of these lanterns were on the ground and did not need any vertical